src/Build/Evaluation/ProjectParser.cs

```diff
@@ -562,46 +528,26 @@ private UsingTaskParameterGroupElement ParseUsingTaskParameterGroupElement(XmlEl
             return parameterGroup;
         }
 
-        /// <summary>
-        /// Parse a UsingTaskBodyElement from the element
-        /// </summary>
-        private ProjectUsingTaskBodyElement ParseUsingTaskBodyElement(XmlElementWithLocation element, ProjectUsingTaskElement parent)
```

Comment:
Same with these, I like it that each parse method is responsible for its own validation checks and interpretation.